_includes/themes/cgal/default.html

```diff
@@ -131,7 +129,7 @@ <h1>{{ page.title }} {% if page.tagline %} <small>{{ page.tagline }}</small>{% e
             &copy; 1995-{{site.time | date: "%Y"}} The CGAL Project. All rights reserved.
             <div style="float: right;">
               <a href="https://cgal.geometryfactory.com/CGAL/Members/wiki/Main_Page" alt="Password protected access to the Developer Wiki">
-                <i class="glyphicon glyphicon-lock"></i>
+                <i class="bi bi-lock-fill fs-5"></i>
```

Comment:
You should search for all occurrences of `glyphicon` in the source code of the web site, to replace them.